app/controllers/buyers/applications_controller.rb

```diff
@@ -24,19 +22,16 @@ def index
     activate_menu :audience, :applications, :listing
 
     @states = Cinstance.allowed_states.collect(&:to_s).sort
-    accessible_services = current_account.accessible_services
-    @services = accessible_services.includes(:application_plans)
     @search = ThreeScale::Search.new(params[:search] || params)
-    @application_plans = current_account.application_plans.stock
-    @stock_and_custom_application_plans = current_account.application_plans.size
+    @application_plans = accessible_plans # .stock
```

Comment:
`features/old/api/plans/customize_application_plan.feature:55` fails with stock

Reply:
We should not list custom plans in this screen, I think your test fails for something else

![image](https://user-images.githubusercontent.com/64276/84996385-e0cba380-b155-11ea-9cb5-629ab1d2a88a.png)

Reply:
sorry @hallelujah but I do not understand since those 2 sentences seem "incompatible" with each other.

> I think your test fails for something else

I'm not sure if we are understanding each other... The test current passes if we do not have the `.stock` (just as you are seeing it now in the review) but fails if the code adds the `.stock`.

You just said that the page should not contain stock, right? If that is the case, the test is correct, and it is passing right now correctly.

> We should not list custom plans in this screen

Okidoki, but why we have this in the `master` branch?
https://github.com/3scale/porta/blob/f7ff1025c4c37570082c4a35623a90dd33d85bc0/app/controllers/buyers/applications_controller.rb#L30

And another question:
Why you don't have the list of services in that screenshot? are you sure that we are on the same page?

Reply:
` @application_plans = current_account.application_plans.stock` means no custom plans
The screenshot is a list of application plans

I got it from `http://provider-admin.example.com/apiconfig/services/{service_id}/application_plans`

Reply:
OK I got confused, it should be the application plan of an account not the defined plans in general
Sorry

Reply:
`http://provider-admin.example.com/apiconfig/services/{service_id}/application_plans` is not the view of this controller 😄 the view of this controller is `http://provider-admin.example.com//buyers/applications`

I know what the scope `stock` is 😉 
https://github.com/3scale/porta/blob/f7ff1025c4c37570082c4a35623a90dd33d85bc0/app/models/plan.rb#L130-L131

My issue now is:
1. How come in the `master` branch we have here the `.stock` but with this change... the test passes without `.stock` but not without it?
2. Should we show custom plans in that specific view or not? According to the `master` branch, I'd say that we do not, but according to the test I'd say that we do... I'm confused 😕

Reply:
Thomas told me that the applications page should definitely show all applications and that if they are on a custom plan is irrelevant 👍